### DISCUSSION TITLE
Proxys with UDP / DNS

### DISCUSSION THREAD

**Discussion by daysus** [engageub/InternetIncome#98] [Q&A]
I bought proxies from fishproxy, but they are all located in the same city. Can you recommend other websites with residential or isp proxies with the UDP and DNS protocol?

**Comment by engageub** [comment]
Hello,
UDP support is provided by very few providers.
There are other providers where you can search online but they are somewhat expensive to recover the investment made.

Here is one https://proxysocks5.com/

Thank you

**Reply by engageub** [reply]
A new list has been updated
https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers

Thank you

**Comment by daysus** [comment]
Thanks for your contribution, I would like to know what monetizers are used for those that are not UDP.

Because there can be ISPs without UDP.

**Reply by engageub** [reply]
Hello,
Except Mysterium in main branch, all other containers are working without UDP.
DNS queries are used to resolve the domain name to get the IP address.

Thank you

**Reply by daysus** [reply]
Oh really? I thought all proxies had to be UDP for honeygain :(. Well they will be cheaper.

**Comment by engageub** [comment]
Yes. There is an option to enable dns via socks5 proxy and is disabled by
default.

Thank you
